learning-bitmark/contributing-to-bitmark/bug-bounty-program.md

```diff
@@ -0,0 +1,63 @@
+# BUG BOUNTY PROGRAM
+
+## **Important:** We only accept bug reports for **bitmarkd** project. Bug reports for web applications or any other project that is not **bitmarkd** will not be accepted and given any reward.
+
+![SUBMIT A BUG](https://docs.google.com/forms/d/e/1FAIpQLSeVzZfd-DDQNuVVDMkwgu4VSxmPnvB6OLo_sw_9CH1w34xoZA/viewform)
+
+#### Bitmark strives to make the Bitmark system safe and secure for everyone. We greatly value the work done by security researchers in improving the security of our products and service offerings. We are committed to working with this community to verify, reproduce, and respond to legitimate reported vulnerabilities. We encourage the community to participate in our responsible reporting process.
+
+##### We offer rewards in Bitcoin:
+| Critical | High | Medium | Low | Note |
+|----------|------|--------|-----|------|
+| 1BTC | 0.5 BTC | 0.2 BTC | 0.1 BTC | 0.01 BTC |
+
+#### Latest bug updates:
+* **2018.11.14**: Congratulations to bughunterboy for finding bugs and earning rewards!
+* **2018.10.11**: Join our Bug Bounty Program and earn rewards now!
+
+
+## How to report a bug
+
+#### Scope:
+![Bitmarkd](https://github.com/bitmark-inc/bitmarkd) is the codebase for the Bitmark blockchain.
+
+##### Bug scope:
+Only bugs in Scope are eligible for the bug bounty. A qualifying bug has to be a danger to the blockchain records, privacy or client operations. These include bugs in:
+1. **Protocol**: Flaws in Protocol design, an example is incentive design flaw in Paper
+2. **Implementation of client**: Any implementation bugs in Bitmarkd to cause bad block, invalid assets, transaction failure bad operations, or crashing of the program
+3. **Cryptography**: Incorrectly implementation of Cryptographic algorithms
+4. **Network Attacks**: Attacks like Sybil attack
+5. Any bugs that we consider important
+
+###### Restrict online low-bandwidth attack to these two nodes only:
+* node-d3.test.bitmark.com
+* node-a3.test.bitmark.com
+
+##### Bugs in third party packages:
+It may or may not be rewarded by Bitmark. In theory, you should claim the bug(s) directly with the third party bounty program; however, we may also reward you for your effort.
+
+##### Bugs out of scope include:
+* Physical access to a user’s device
+* Social engineering
+* MITM attacks
+* DDoS / Fuzzing / High-Bandwidth Attack
+
+##### Submit
+* Submit your report to: ![Bug Bounty Submission Form](https://docs.google.com/forms/d/e/1FAIpQLSeVzZfd-DDQNuVVDMkwgu4VSxmPnvB6OLo_sw_9CH1w34xoZA/viewform)
+
+##### Collect rewards
```

Comment:
This doesn't actually say how to collect rewards. You should say what time frame (or range of times) that you'll get back to reporter and how you'll contact them. (e.g.: "We'll usually respond to you within 2-3 business days, depending on the severity of the bug, by mailing the email address you supplied us. At the same time if the bug was of note and in scope, we'll make a payment your cryptocurrency address.") Or whatever's actually appropriate for your procedure and what information you collect to get in contact with (or just pay) the person on your form.